Firestore/src/FirestoreClient.php

```diff
@@ -159,6 +159,14 @@ public function __construct(array $config = [])
         );
     }
 
+    /**
+     * Todo: Remove before committing to main.
+     */
+    public function myCustomFunction()
+    {
+        print('Do NOT commit to main.');
+    }
+
```

Comment:
This is leading to BC breaks: [Log](https://github.com/googleapis/google-cloud-php/actions/runs/5643607796/job/15285783686?pr=6481)
```suggestion
```